Subject: [PATCH] dhcpcd: Fix build error with musl.

Re-organize the header includes in if-linux.c to fix error as follows:
In file included from if-linux.c:49:
path/usr/include/netinet/if_ether.h:112:8: error: redefinition of 'struct ethhdr'

Signed-off-by: Zheng Ruoqin <zhengrq.fnst@cn.fujitsu.com>
---
 src/if-linux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/if-linux.c b/src/if-linux.c
index fd47278..bfa878b 100644
--- a/src/if-linux.c
+++ b/src/if-linux.c
@@ -37,6 +37,7 @@
 #include <linux/if_addr.h>
 #include <linux/if_link.h>
 #include <linux/if_packet.h>
+#include <netinet/if_ether.h>
 #include <linux/if_tun.h>
 #include <linux/if_vlan.h>
 #include <linux/filter.h>
@@ -46,7 +47,6 @@
 
 #include <arpa/inet.h>
 #include <net/if.h>
-#include <netinet/if_ether.h>
 #include <netinet/in_systm.h>
 #include <netinet/in.h>
 #include <net/route.h>
-- 
2.7.4

